### PR TITLE
Adding page on `UiSettings` adjustments

### PR DESCRIPTION
**Pull request by langsmith**
Resolves https://github.com/mapbox/android-docs/issues/99 by adding a page on adjusting the Maps SDK's `UiSettings` such as gestures, attribution tint, etc.

screencapture-localhost-8080-android-maps-overview-ui-settings-2019-11-19-15_37_11

### CODE REVIEW

File: src/pages/maps/overview/ui-settings.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+`}
+
+/>
+}}
+
+Changes to gesture recognition, the compass, the Mapbox logo, and attribution can be made once you have the `UiSettings` object. Changes made to `UiSettings` are immediately reflected on the map.
+
+You can also configure the UI settings with [the Maps SDK's XML attributes](/android/maps/overview/#mapview-xml-attributes) instead of the `UiSettings` class' methods.
+
+All the UI settings also have "get" methods, which helps you find out the current settings such as whether the compass is enabled, what the logo's margins are, or what the custom compass image is.
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/ui-settings.md]:
```suggestion
All the UI settings also have "get" methods that allow you to see the current settings (for example, if the compass is enabled, what the logo's margins are, or what the custom compass image is).
```

File: src/pages/maps/overview/ui-settings.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+`}
+
+/>
+}}
+
+## Gestures
+
+The Mapbox Maps SDK for Android uses [the Mapbox Gestures for Android library](/android/maps/overview/gestures/) to detect gestures.
+
+Adjust or completely disable zoom, tilt, scroll, rotate, quick zoom, and/or double tap gesture recognition. Disabling map gestures doesn't affect whether you can change the camera position programmatically.
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/ui-settings.md]:
```suggestion
Adjust or completely disable zoom, tilt, scroll, rotate, quick zoom, and double tap gesture recognition. Disabling map gestures doesn't affect whether you can change the camera position programmatically.
```

File: src/pages/maps/overview/ui-settings.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+Here's how you would disable the ability to increase the zoom level by double tapping on the map:
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/ui-settings.md]:
```suggestion
For example, here's how you would disable the ability to increase the zoom level by double tapping on the map:
```

File: src/pages/maps/overview/ui-settings.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+`}
+
+/>
+}}
+
+See the [map camera position documentation](/android/maps/overview/camera/#camera-position) for more information about how gestures can affect camera position.
+
+## Compass
+
+The Maps SDK contains a compass image, which is appears on the map when the map is rotated in any direction. The compass _doesn't_ represent the physical device's relation to the magnetic north pole. The compass image rotates based on the on-screen map rotation.
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/ui-settings.md]:
```suggestion
The Maps SDK contains a compass image, which appears on the map when the map is rotated in any direction. The compass _doesn't_ represent the physical device's relation to the magnetic north pole. The compass image rotates based on the on-screen map rotation.
```

File: src/pages/maps/overview/ui-settings.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+}}
+
+
+## Logo
+
+Please read [Mapbox's attribution policy](/android/maps/overview/#attribution) for information on displaying the Mapbox wordmark logo.
+
+The logo's properties such as margins and gravity can be adjusted with XML or the `UiSettings` object. 
+
+## Attribution
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/ui-settings.md]: Could consider combining the Logo and Attribution sections.

Comment (langsmith) [reply src/pages/maps/overview/ui-settings.md]: Agree. Done and pushed.

**Comment by langsmith** [comment]
Will not merge until @danswick gives the `:greenlight:`

**Pull request merged**